ui/src/app/views/workspace-list/component.html

```diff
@@ -1,6 +1,19 @@
 <div class="area-block">
   <div class="table-header">
     <h3 class="table-header-descriptor">Your Workspaces</h3>
+    <div class="alert alert-warning" *ngIf="twoFactorShow">
+      <div class="alert-items" style="width: 18rem;">
+        <div class="alert-item static">
+          <span class="alert-text">
+            If you have not already, please enable two factor authentication
+            <a class="warning-link" href="https://myaccount.google.com/security">here</a>.
+          </span>
+          <div class="alert-icon-wrapper">
+            <clr-icon class="alert-icon" shape="times" (click)="twoFactorShow = false;"></clr-icon>
```

Comment:
Hmm, seems like we need to have something much stickier to dismiss this. Is there a plan for this?

One idea: look at user creation time, don't show this if the account is 3(?) days old? I forget when they would have been disabled.

Reply:
I can add something like that, sure!

Reply:
Also, we may want get rid of the "x" button if we go with that approach. I think it will be annoying if they X out, only to have it reappear when they next visit that page, or refresh their browser.

Reply:
I don't think we actually store user creation time right now, actually? I could add that to the API, but I think that as a followup makes more sense?

Reply:
Sure, that sounds fine. I'd recommend making it undismissable in any case.

Reply:
Yeah, its not super obtrusive too, so its probably fine to make it undismissable.